### DISCUSSION TITLE
Which MONAI model to use for liver segmentation

### DISCUSSION THREAD

**Discussion by gokceay** [Project-MONAI/MONAILabel#1506] [General]
Hi, 

There are many models that segments liver in MONAI. What I want to ask is which one should I use to get the most accurate and correct liver segmentation? To increase the accuracy and smoothness of the related organ, do I have to retrain the network by resampling the dataset to a lower spacing value? If so, how should I retrain? Is there any specific tutorial about it? 

Thanks in advance

**Comment by diazandr3s** [comment]
Hi @gokceay,

Very good question.
I'd suggest you try the default Segmentation model in the Radiology app: https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/radiology#segmentation

It is not only for the Liver segmentation, it actually segments [20+ organs](https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/segmentation.py#L37-L63). But it works great and you could tweak it to only show the liver if needed.

`
To increase the accuracy and smoothness of the related organ, do I have to retrain the network by resampling the dataset to a lower spacing value?`

That's correct. Be mindful that this change will require more GPU memory to retrain the model.


Hope this helps,
